### PR TITLE
suggest getting gcc, g++ from conda

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,18 +2,15 @@
...
-
 **Optional: Installing an older compiler**.
-If you want to go the extra mile, you can install gcc-9, g++-9, as this is the minimum required version.
+If you want to go the extra mile, you can install gcc 9, as this is the minimum required version.
 This will help you make sure the code you write works on these compilers.
-Note that to do this you might need to do something along the lines of https://askubuntu.com/a/26518
-At the time of this writing, I got some errors when compiling cublasLt as it was using a custom g++10
-
+To do this, add `gxx=9` to `pytorch-dev.yaml` file. At the time of this writing, building with conda
+gxx=9 requires cuda-nvcc to be installed from conda as well.
```
Comment (lezcano) [inline README.md:9]: Does this currently work?

Comment (lezcano) [reply README.md:9]: The best way to make sure all this works would be to pull a fresh paperspace machine with an A100 and see if it builds properly. I can give you access to one and you can see if it works as expected.

Comment (isuruf) [reply README.md:9]: It does work on qgpu3. At least pytorch gets build successfully.

**Review by lezcano — APPROVED** [review]
The previous comment was not particularly informative, so let's take this one that, at last, works.

**Pull request merged**